### PR TITLE
feat(SWA-11): Add stepper for the submission flow

### PR DESCRIPTION
**Pull request by Serge0n**
Jira Ticket:ticket: : [SWA-11](https://artsyproduct.atlassian.net/browse/SWA-11)

This PR adds  `SubmissionStepper` for the new submission flow. Also added redirect from `/consign/submission2` to `/consign/submission2/artwork-details`

**VIDEO:clapper: :**

https://user-images.githubusercontent.com/55637696/135874371-0beada1f-2849-4e01-847a-a50673410bbb.mov

### CODE REVIEW

**Review by iskounen — APPROVED** [review]

**Review by iskounen — APPROVED** [review]

**Review by kiryl-zubarau — APPROVED** [review]

File: src/v2/Apps/Consign/Components/SubmissionStepper.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            disableNavigation
+            autoScroll
+          >
+            {submissionFlowSteps.map(step => (
+              <Step name={step} key={step} />
+            ))}
+          </Stepper>
+        </Box>
+      </Media>
+      <Media greaterThan="sm">
```
Comment (damassi) [inline src/v2/Apps/Consign/Components/SubmissionStepper.tsx:39]: @Serge0n - Its somewhat hard to read / understand seeing `between="xs, md"` and `greaterThan="sm". Does this mean: "at sm" and "at md"? 

Better yet, any chance we can refactor this to follow the more common pattern of:

```
<Media at=''xs'>
  ...
 </Media>
 <Media greaterThan='xs'>
    ...
 </Media>
 ```

Which simply means: "on phones render A; on desktops render B"

File: src/v2/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/ArtworkDetails.tsx [hunk 2]
```diff
@@ -1,6 +1,12 @@
 import React, { FC } from "react"
 import { Box } from "@artsy/palette"
+import { SubmissionStepper } from "v2/Apps/Consign/Components/SubmissionStepper"
 
 export const ArtworkDetails: FC = () => {
-  return <Box mt={50}>Artwork Details step</Box>
+  return (
+    <>
+      <SubmissionStepper currentStep="Artwork Details" />
+      <Box mt={50}>Artwork Details step</Box>
```
Comment (damassi) [inline src/v2/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/ArtworkDetails.tsx:9]: We should never step out of our [spacing scale values](https://palette.artsy.net/atoms/spacing/), and instead round up or down to the closest available option.

File: src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/ContactInformation.tsx [hunk 3]
```diff
@@ -1,6 +1,12 @@
 import React, { FC } from "react"
 import { Box } from "@artsy/palette"
+import { SubmissionStepper } from "v2/Apps/Consign/Components/SubmissionStepper"
 
 export const ContactInformation: FC = () => {
-  return <Box mt={50}>Contact Information step</Box>
+  return (
+    <>
+      <SubmissionStepper currentStep="Contact Information" />
+      <Box mt={50}>Contact Information step</Box>
```
Comment (damassi) [inline src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/ContactInformation.tsx:9]: ditto

File: src/v2/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/UploadPhotos.tsx [hunk 4]
```diff
@@ -1,6 +1,12 @@
 import React, { FC } from "react"
 import { Box } from "@artsy/palette"
+import { SubmissionStepper } from "v2/Apps/Consign/Components/SubmissionStepper"
 
 export const UploadPhotos: FC = () => {
-  return <Box mt={50}>Upload Photos step</Box>
+  return (
+    <>
+      <SubmissionStepper currentStep="Upload Photos" />
+      <Box mt={50}>Upload Photos step</Box>
```
Comment (damassi) [inline src/v2/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/UploadPhotos.tsx:9]: ditto

**Review by damassi — COMMENTED** [review]
@Serge0n - there's a few changes that need to be made in a follow-up PR, but otherwise looks good.

File: src/v2/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/UploadPhotos.tsx (hunk 4 above)
Comment (Serge0n) [reply src/v2/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/UploadPhotos.tsx:9]: fixed in following PR https://github.com/artsy/force/pull/8545

**Pull request merged**